server/src/domain/sql_migrations.rs

```diff
@@ -460,30 +459,81 @@ async fn migrate_to_v3(pool: &DbConnection) -> anyhow::Result<()> {
     Ok(())
 }
 
+async fn migrate_to_v4(pool: &DbConnection) -> anyhow::Result<()> {
+    let builder = pool.get_database_backend();
+    // Make emails and UUIDs unique.
+    pool.execute(
```

Comment:
Note that I can't use `replace_column`, since you can't create column with "unique" constraints in sqlite. You get the same effect by creating a unique index, though.